root/entity/Subscribers.js

```diff
@@ -45,108 +48,116 @@ const Subscribers = ({
     >
       <h2>{l('Subscribers')}</h2>
 
-      {(publicEditors.length || (privateEditors > 0)) ? (
+      {isSpecialPurpose(entity) ? (
+        <p>
+          {l('This is a special entity and does not support subscriptions.')}
```

Comment:
We use "special purpose" in most strings, so just for consistency
```suggestion
          {l('This is a special purpose entity and does not support subscriptions.')}
```